python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -48,6 +48,7 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         ('Configuration', 'gangadir', gangadir_for_test),
         ('Configuration', 'user', 'testframework'),
         ('Configuration', 'repositorytype', 'LocalXML'),
+        ('Configuration', 'UsageMonitoringMSG', False),  # Turn off spyware
```

Comment:
If you point it to a non existent server in the config the messages should silently drop. (I'll add an issue to add a test for this).
It would make sense to leave it on for the parallel stuff imo. It should be safe, but I'd like to catch any problems in testing :)